layouts/partials/main-page.html

```diff
@@ -242,28 +242,6 @@ <h2>Join Our Community</h2>
         </svg>
       </a>
     </span>
-    <span class="community-icon">
```

Comment:
Don't remove this. Replace url with `https://kiwiirc.com/nextclient/#irc://irc.libera.chat/#rizin`